controllers/controllers.js

```diff
@@ -52,11 +53,27 @@ function getCommentsByArticleId(req, res, next) {
     });
 }
 
+function postCommentByArticleId(req, res, next) {
+  const { article_id } = req.params;
+  const { body: newComment } = req;
+  if (!newComment.username || !newComment.body) {
+    next({ status: 400, msg: "invalid body" });
```

Comment:
If you take a look at the seed,
` CREATE TABLE comments (
        comment_id SERIAL PRIMARY KEY,
        body VARCHAR NOT NULL,
        article_id INT REFERENCES articles(article_id) NOT NULL,
        author VARCHAR REFERENCES users(username) NOT NULL,
        votes INT DEFAULT 0 NOT NULL,
        created_at TIMESTAMP DEFAULT NOW()
      );`);`

You don't need to invoke next here as psql error handler should take care of it. If you want a custom message over 'bad request' you should really do a promise.reject.